ios_app_clip/test/widget_test.dart

```diff
@@ -0,0 +1,30 @@
+// This is a basic Flutter widget test.
+//
+// To perform an interaction with a widget in your test, use the WidgetTester
+// utility that Flutter provides. For example, you can send tap and scroll
+// gestures. You can also use WidgetTester to find child widgets in the widget
+// tree, read text, and verify that the values of widget properties are correct.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import 'package:ios_app_clip/main.dart';
+
+void main() {
+  testWidgets('Counter increments smoke test', (WidgetTester tester) async {
```

Comment:
If you revise the main page of the app, feel free to replace this with a stub test. It does need to have one, though, and get listed in `/tool/travis_flutter_script.sh`

Reply:
Actually, consider adding it to `/tool/travis_ios_script.sh` as well.

Reply:
ok, I think I did. I'm not sure I understood the travis_ios_script. Did you mean whether I had an XCTest for this?

Reply:
No, I just want it to build. We have some full integration tests for the add-to-app Android apps, but haven't had bandwidth to do anything for iOS.